package.json

```diff
@@ -26,8 +26,21 @@
   },
   "babel": {
     "presets": [
-      ["@babel/preset-env",{"targets": {"esmodules": true}}],
-      ["@babel/preset-react",{"runtime": "automatic"}]]
+      [
+        "@babel/preset-env",
+        {
```

Comment:
I would like to keep this format same as before. What was the reason for re-formatting this?